src/app/templates/activity/MoneyDiffView.tsx

```diff
@@ -37,7 +39,7 @@ const MoneyDiffView = memo<MoneyDiffViewProps>(({ assetId: assetSlug, diff, pend
         <InUSD volume={diffBN.abs()} assetSlug={assetSlug}>
           {usdVolume => (
```

Comment:
refactor&rename `InUSD` component to let it return `selectedFiatCurrency` symbol.  
`+` alight all names across this feature